daemons/controld/controld_te_events.c

```diff
@@ -177,17 +186,38 @@ update_failcount(xmlNode * event, const char *event_node_uuid, int rc,
 
         /* Update the fail count, if we're not ignoring failures */
         if (!ignore_failures) {
-            attr_name = pcmk__failcount_name(rsc_id, task, interval_ms);
-            update_attrd(on_uname, attr_name, value, NULL, is_remote_node);
-            free(attr_name);
+            fail_pair = calloc(1, sizeof(pcmk__attrd_query_pair_t));
```

Comment:
assert non-null, static analysis is pickier about that these days

Reply:
Fixed.